viewer-prototype/src/browser/trace-viewer/components/utils/filtrer-tree/utils.tsx

```diff
@@ -10,15 +10,15 @@ const entryToTreeNode = (entry: Entry) => ({
         children: []
     } as TreeNode);
 
-export const listToTree = (list: Entry[]): TreeNode[] => {
+export const listToTree = (list: Entry[], rootId: number): TreeNode[] => {
```

Comment:
You could add a default value for rootId. Also, why do you need the rootId parameter? I see it's just for the current workaround for empty rows... oh well!